cherrypy/tutorial/.dockerignore

```diff
@@ -0,0 +1,2 @@
+ENV
```

Comment:
why ignore it? there's not env/ENV here

Reply:
It was an ENV directory in my system when I first built the container I did not want to pollute the container with it.

Reply:
I see, but now it's not needed I think